about.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./about.css" />
+    <script src="./js/script.js" defer></script>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>Latest Routing Technology | About Page</title>
+  </head>
+  <body>
+    <header id="header">
+      <div class="top-navbar">
+        <ul class="top-navbar-list container flex-center">
+          <li>
+            <a href="#"><i class="fa fa-twitter"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa fa-facebook-f"></i></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
+      <div class="main-navbar flex-center">
+        <div class="navbar-content container flex-center">
+          <div class="logo">
+            <a href="index.html"></a>
+          </div>
+          <nav class="navbar flex-center">
+            <div class="menu">
+              <i class="fa fa-align-justify"></i>
+            </div>
+            <ul class="nav-list flex-col">
+              <li class="nav-link">
+                <a class="about" href="about.html">About</a>
+              </li>
+              <li class="nav-link"><a href="#mission">Mission</a></li>
+              <li class="nav-link"><a href="#team">Team</a></li>
+              <li class="nav-link"><a href="#partner">Partner</a></li>
+              <li class="nav-link"><a href="index.html">Home</a></li>
+              <li class="nav-link">
+                <a class="tech" href="#mission">H.Tech</a>
+              </li>
+            </ul>
+            <div class="close">
+              <i class="fa fa-times"></i>
+            </div>
+          </nav>
```

Comment:
- [ ] Currently, the hamburger icon is visible both on the mobile and desktop view. As per design, it's supposed to only be visible in the mobile view, while the desktop one has a wide navigation menu at the top of the page. Please make sure to use media queries and fix the views.

Reply:
Done